src/main/resources/configCivcraft.yml

```diff
@@ -9487,7 +9472,7 @@ recipes:
     input:
       emeralds:
         material: EMERALD
-        amount: 5
+        amount: 10
```

Comment:
I actually agree with Dan here. This would further empower people that can already afford high tier enchants and make it harder for the smaller fish to get lower level enchants for themselves. Barrier to entry is the biggest issue civraft is facing right now IMO.

We want people to feel like they can do something without being forced into big states, while still giving large states an advantage. Both of those goals are obviously conflicting, making it hard to balance. I don't think this change helps though.